impeller/entity/contents/content_context.h

```diff
@@ -408,7 +408,7 @@ class ContentContext {
   ///
   // TODO(bdero): Remove this setting once StC is fully de-risked
   //              https://github.com/flutter/flutter/issues/123671
-  static constexpr bool kEnableStencilThenCover = true;
+  static constexpr bool kEnableStencilThenCover = false;
```

Comment:
@bdero this only works if I turn stencil and cover off (of course inner and outer are reversed though)

Reply:
Taking a look at this now